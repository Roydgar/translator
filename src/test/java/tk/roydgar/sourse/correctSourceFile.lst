----Code----
;program1as

data SEGMENT

ident1	EQU	'10#-110'
ident2	EQU	'30'

data ENDS

code SEGMENT
		ASSUME cs:code, ds:data

begin:
	push ebp
	mov ebp, esp
	xor eax, eax
	mov esp, ebp
	pop ebp
	ret 0
	mov ax, 4c00h
	int 21h
code ENDS
	END begin