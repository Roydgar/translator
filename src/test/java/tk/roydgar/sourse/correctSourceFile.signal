program program1as;
(*******)

Const ident1 = 10#-110;
   (***----***)
const ident2= 30;

BEGIN

END.
