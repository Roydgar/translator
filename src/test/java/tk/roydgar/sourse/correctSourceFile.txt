program program1as;
(*******)

BEGIN
   Const ident1 = 10#110;
   (***----***)
   const ident2= -30;
END.
