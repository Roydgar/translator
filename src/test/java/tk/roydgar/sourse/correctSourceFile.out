   1    0        301    program
   1    7        401    program1as
   1   17          3    ;
   4    0        302    begin
   5    3        301    const
   5    8        402    ident1
   5   14          0    =
   5   15        501    10#110
   5   21          3    ;
   7    3        301    const
   7    8        403    ident2
   7   14          0    =
   7   15          2    -
   7   16        502    30
   7   18          3    ;
   8    0        303    end
   8    3          5    .
